subsys/net/l2/ieee802154/ieee802154_priv.h

```diff
@@ -63,7 +63,7 @@ int ieee802154_wait_for_clear_channel(struct net_if *iface);
  *        This function has side effects and must be called before each individual transmission
  *        attempt.
  *
- *        This function checks for and supports both, software and hardware acknowlegement,
+ *        This function checks for and supports both, software and hardware acknowledgement,
```

Comment:
@ndrs-pst  can you please fix that?

Reply:
we can fix the spelling file later, both versions of the spelling are correct it seems.